src/client/unified-form/validators/cover-tab.ts

```diff
@@ -40,13 +40,11 @@ export function isCoverTabEmpty(data:any) {
 	const nameSection = get(data, 'nameSection', {});
 	const authorCreditEditor = get(data, 'authorCreditEditor', {});
 	const ISBN = get(data, 'ISBN', {});
-	const identifierEditor = get(data, 'identifierEditor', {});
 	return nameSection.name?.length === 0 &&
     nameSection.sortName?.length === 0 &&
     !nameSection.language &&
     nameSection.disambiguation?.length === 0 &&
     size(authorCreditEditor) === 1 &&
     !authorCreditEditor.n0?.author &&
-    size(identifierEditor) === 0 &&
```

Comment:
I don't think this should be removed.
This function checks if the cover tab is empty, which should indeed be false if the user has input an ISBN or any other identifier.